test/test_dispatch.py

```diff
@@ -277,27 +285,61 @@ def test_impl_only(self):
 catchall: impl_t_t :: (Tensor _0) -> (Tensor _0) [ boxed unboxed ]
 ''')
 
+    def test_computed_table(self):
+        state, table = self.commute("foo", [
+            # m.def("foo", [](const Tensor & x) { return x })
+            lambda m: m.def_name_t_t("foo"),
+            # m.impl("foo", torch::kCPU, [](const Tensor & x) { return x })
+            lambda m: m.impl_t_t("foo", "cpu"),
+            # m.impl("foo", torch::kCUDA, [](const Tensor & x) { return x })
+            lambda m: m.impl_t_t("foo", "xla"),
+            # m.impl("foo", torch::kAutograd, [](const Tensor & x) { return x })
+            lambda m: m.impl_t_t("foo", "autograd"),
+            # m.impl("foo", torch::kAutogradCPU, [](const Tensor & x) { return x })
+            lambda m: m.impl_t_t("foo", "autogradcpu")
+        ])
+        self.assertExpectedInline(state, '''\
+name: test::foo
+schema: test::foo(Tensor _0) -> (Tensor _0)
+debug: registered at /dev/null:0
+alias analysis kind: CONSERVATIVE
+CPU: impl_t_t :: (Tensor _0) -> (Tensor _0) [ boxed unboxed ]
+XLA: impl_t_t :: (Tensor _0) -> (Tensor _0) [ boxed unboxed ]
+AutogradCPU: impl_t_t :: (Tensor _0) -> (Tensor _0) [ boxed unboxed ]
+Autograd[alias]: impl_t_t :: (Tensor _0) -> (Tensor _0) [ boxed unboxed ]
+catchall: default_def_name_t_t :: (Tensor _0) -> (Tensor _0) [ boxed unboxed ]
+''')
+        # computed dispatch table is too big to use assertExpectedInline,
+        # so we only assert on a few entries we're interested in.
```

Comment:
My preference is for us to instead specify a list of keys that we are interested in seeing in the computed dispatch table, and then use an expect test on it. That will make it easier for us to do wide-sweeping refactors to the dump format in the future.